src/components/ui/TripCard/TripCardAccordion.tsx

```diff
@@ -0,0 +1,64 @@
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import longArrow from '../../../assets/Long_arrow.svg';
+import acIcon from '../../../assets/AcIcon.svg';
+import nonAcIcon from '../../../assets/NonAcIcon.svg';
+import seatIcon from '../../../assets/SeatIcon.svg';
+import sleeperIcon from '../../../assets/SleeperIcon.svg';
+import Stack from '@mui/material/Stack';
+import { TripAccordionStyled } from './TripCard.styled';
+
+interface ITripCardAccordion {
+    busType: string;
+    seatType: string;
+}
+export const TripCardAccordion = ({
+    busType,
+    seatType,
+}: ITripCardAccordion) => {
+    return (
+        <TripAccordionStyled className="summary">
+            <AccordionSummary
+                expandIcon={<ArrowDropDownIcon />}
+                aria-controls="panel1-content"
+                id="panel1-header"
```

Comment:
not a good way to add magic numbers like panel1

Reply:
Ok